docs/content/_terraform/index.md

```diff
@@ -2,125 +2,68 @@
 title: Terraform
 ---
 
-# Terraform
+# About
 
-[Terraform](https://www.terraform.io/) is the recommended infrastructure-as-code (IAC) tool in the BYU ecosystem. IAC
-tools enables developers automated the creation, modification, and destruction of infrastructure. It also allows for 
-better documentation and versioning of infrastructure.
+## What is Terraform?
+For our purposes, [Terraform](https://github.com/hashicorp/terraform) is a declarative way to provision AWS resources. 
+It can be used for infrastructure or deployable applications.
 
-## Terraform Modules
+To use Hashicorp's own description, [Terraform](https://github.com/hashicorp/terraform) enables you to safely and 
+predictably create, change, and improve infrastructure. It is an open source tool that codifies APIs into declarative 
+configuration files that can be shared amongst team members, treated as code, edited, reviewed, and versioned.
 
-These modules simplify and standardize the creation of application infrastructure. You reference these modules from 
-`.tf` files inside your project repo via versioned URL.
+You can think of [Terraform](https://github.com/hashicorp/terraform) as OIT's successor to 
+[handel](https://github.com/byu-oit/handel) and [handel-codepipeline](https://github.com/byu-oit/handel-codepipeline). 
```

Comment:
Nowadays we're suggesting that people move from `handel` to Terraform and from `handel-codepipeline` to GitHub Actions.

That is, we're no longer suggesting that people spin up AWS CodePipelines using Terraform.

Reply:
Fixed